pkg/evpn/evpn.go

```diff
@@ -34,11 +34,8 @@ type Server struct {
 	pb.UnimplementedSviServiceServer
 	pb.UnimplementedLogicalBridgeServiceServer
 	pb.UnimplementedBridgePortServiceServer
-	Bridges    map[string]*pb.LogicalBridge
-	Ports      map[string]*pb.BridgePort
-	Svis       map[string]*pb.Svi
-	Vrfs       map[string]*pb.Vrf
 	Pagination map[string]int
+	ListHelper map[string]bool
```

Comment:
@glimchb since `ListHelper` is used as a set, it is more optimal to use `struct{}` as a value to save up 1 byte per entry (empty struct occupies 0 bytes).